File: Part4/C/Function_MissingParameter.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    char userInput[ 100 ];
    
    printf( "Type 1 and press enter.\n" );
    fgets( userInput, sizeof( userInput ), stdin );
    
    // Function requires 1 parameter, but we're passing 0
    int enteredInteger = atoi();
    
    printf( "The user entered the integer %d\n", enteredInteger );
    
    return 0;
}

File: Part4/Go/Function_MissingParameter.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var userInput string

	fmt.Println( "Type 1 and press enter." )
	fmt.Scanln( &userInput )

	// Function requires 1 parameter, but we're passing 0
	enteredInteger, _ := strconv.Atoi()

	fmt.Printf( "The user entered the integer %d\n", enteredInteger )
}
